ui/src/views/auth/Login.vue

```diff
@@ -189,8 +189,8 @@ export default {
             }
             Login(loginParams)
               .then((res) => this.loginSuccess(res))
-              .catch(err => this.requestFailed(err))
-              .finally(() => {
+              .catch(err => {
+                this.requestFailed(err)
                 state.loginBtn = false
```

Comment:
this would set loginBtn to false on error?

Reply:
@rhtyd I think it is correct because the goal is to only re-enable the button when it gets an error to perform click/submit again.

Reply:
What he said ^^

Reply:
Thanks for explaining, I thought setting that to 'true' would mean enabling the button.